src/DoctrineBridge/Router/EntityIdResolver.php

```diff
@@ -4,7 +4,7 @@
 
 use Doctrine\Common\Persistence\ManagerRegistry;
 use Doctrine\ORM\Mapping\MappingException;
-use Iltar\HttpBundle\Exception\UncallableMethodException;
+use Iltar\HttpBundle\DoctrineBridge\IdFromEntityInterface;
```

Comment:
I would expect this interface to not live in DoctrineBridge, as the goal of the interface is to allow other implementations (which don't depend on Doctrine).